common/components/controllers/CorporateHackathonController.jsx

```diff
@@ -298,180 +289,228 @@ class CorporateHackathonController extends React.PureComponent<{||}, State> {
+                  src="https://d1agxr2dqkgkuy.cloudfront.net/img/bill-melinda-gates-foundation.png"
+                  alt="Bill and Melinda Gates Foundation logo"
+                ></img>
+              </div>
+              {this._renderCurrentSponsors()}
+            </div>
           </div>
         </div>
-        <div className="corporate-sponsorship-saying corporate-section col-12">
-          <h1>What People Are Saying</h1>
+        {this._renderContactSponsor()}
+
+        <div className="corporate-sponsorship-saying corporate-bg-light col-12">
+          <h1>What our partners are saying.</h1>
+          <h2>Feedback from partnering organizations.</h2>
+
           <div className="carousel-testimonial-root">
             <TestimonialCarousel
               category="hackathon-highlights"
               interval={600000}
             />
           </div>
-        </div>
-        <div className="corporate-sponsorship-impact corporate-section col-12">
-          <h1>Impact Stories</h1>
-          <div className="carousel-blog-root">
-            <BlogCarousel tag="hackathon-highlights" interval={600000} />
-          </div>
-          <p className="headline2 font-weight-normal">
+          <p className="headline2">
             Read about other previous events in our{" "}
-            <a
-              className="headline2 font-weight-normal"
-              href={window.BLOG_URL}
-              target="_blank"
-            >
-              blog
-            </a>
-            , or other events that are coming up{" "}
-            <a className="headline2 font-weight-normal" href="/events">
-              here
-            </a>
-            .
+            <a href={window.BLOG_URL}>blog</a>, or other events that are coming
+            up <a href="/events">here</a>.
           </p>
         </div>
       </React.Fragment>
     );
   }
-  _renderContact(): React$Node {
+  _renderContactHackathon(): React$Node {
     return (
       <React.Fragment>
-        <div className="corporate-section corporate-contact col-12">
-          <JumpAnchor id="contact" />
-          <h1>Take The First Step!</h1>
-          <h2>
-            To receive more information about becoming a corporate partner,
-            complete and submit the form below.
-          </h2>
-          <ContactForm showInterests={true} />
+        <div className="corporate-contact col-12">
+          <JumpAnchor id="contact-hackathon" />
+          <h1>Interested in hosting your own hackathon?</h1>
+          <h2>Get in touch to discuss your next tech-for-good event.</h2>
+          <ContactForm interest_hackathon={true} />
+          <h4>
+            Not ready yet? Learn more:{" "}
+            <a
+              href={cdn.document(
+                "2021+DemocracyLab+Corporate+Hackathon+Prospectus.pdf"
+              )}
+            >
+              Corporate Tech-for-Good Hackathons PDF{" "}
+              <i className={Glyph(GlyphStyles.PDF, GlyphSizes.X1)}></i>
+            </a>
```

Comment:
This is duplicated at line 402, can we break this out into a shared block?

Reply:
Makes sense. Since they're both reused, I broke out both the Hackathon and Sponsorship prospectus PDFs into shared blocks. (Also this helped me catch a mistaken use of Hackathon in the Sponsorship tab, so two for one!)